prow/jobs/custom/infra.yaml

```diff
@@ -303,46 +303,6 @@ periodics:
 
 postsubmits:
   knative/infra:
-  - name: post-knative-infra-image-push
```

Comment:
If you are removing this, I am worried about the comment in the `push-all.sh` file which says that prow will crash out without it?

Reply:
look at the first line of that script.

```
# prow-tests image crashes prow job, don't build it here
EXCLUDED="prow-tests"
```

Reply:
That should read, prow-tests image is huge and needs a separate job for it